### DISCUSSION TITLE
Flows webhooks stopped working in 9.23

### DISCUSSION THREAD

**Discussion by luciandavidescu** [directus/directus#17689] [Archive]
I had this webhook running hourly for a few weeks now, as soon as I updated to 9.23.1 it silently fails.

The request still works fine with Postman, returning a 200 status and the expected response. I also tested the same flow on the community cloud, that still runs on 9.22.4, and it works there.

I tried other requests on my installation, some do work while some not, can't figure the reasons, so I'm not sure if it's a bug or I'm doing something wrong.

_This is what it does now:_
![bad](https://user-images.githubusercontent.com/26859276/222653619-c9d1ee64-c9e5-4f8d-b15d-bf3140fe1b5f.jpg)

_This is how it used to run:_
![good](https://user-images.githubusercontent.com/26859276/222653644-51cfd533-62a8-4ba2-9016-7e1759a5b572.jpg)

**Comment by luciandavidescu** [comment]
Update: this most likely has to do with wildcard SSL certificates on the endpoint, switched to a individual subdomain certificate and it now works again.

**Reply by huynkhai0505** [reply]
Hi, can you pls explain more about how can you `switched to a individual subdomain certificate` ? thanks

**Comment by luciandavidescu** [comment]
Had a wildcard ssl certificate before (*.example.com) so I generated a
specific one ***@***.***)
